src/sentry/web/urls.py

```diff
@@ -135,6 +135,13 @@ def init_all_applications():
     ),
     url(r'^api/store/schema$', api.StoreSchemaView.as_view(), name='sentry-api-store-schema'),
 
+    # Frontend client config
+    url(
+        r'^api/client-config/?$',
+        api.ClientConfigView.as_view(),
+        name='sentry-api-client-config'
```

Comment:
Would be good to have tests for this new endpoint. Covering the normal and superuser scenarios would be good.

Reply:
Added tests.